server/controllers/articles.controller.js

```diff
@@ -11,7 +11,17 @@ exports.getArticle = (req,res,next) => {
 }
 
 exports.getArticles = (req,res,next) => {
-    fetchArticles().then((articles)=>{
+    fetchArticles().then((articles) => {
         res.status(200).send({articles})
     })
+}
+
+exports.getComments = (req,res,next) => {
+    const {article_id} = req.params
+    let promises = [fetchComments(article_id),fetchArticle(article_id)]
```

Comment:
Nice to see you remember how to use Promise.all but what is the purpose? All you return is the comments, not the article, so we shouldn't need to fetch any article here.